### ISSUE TITLE
Error When installing to RasperryPi 3

### ISSUE THREAD

**Issue by tigeryell99** [eshapiro42/OctoPrint-Wyze#3]
I am getting the following error when trying to install this plugin to OctoPi running on a RaspberryPi 3:
Does wyze-sdk run on RaspberryPI? 

```
Installing plugin from https://github.com/eshapiro42/OctoPrint-Wyze/archive/main.zip...
/home/pi/oprint/bin/python3 -m pip --disable-pip-version-check install file:///tmp/tmplnlxaz9f/OctoPrint-Wyze-main.zip --no-cache-dir
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Processing /tmp/tmplnlxaz9f/OctoPrint-Wyze-main.zip
Requirement already satisfied: OctoPrint in /home/pi/oprint/lib/python3.7/site-packages (from OctoPrint-Wyze==0.1.4) (1.7.3)
ERROR: Could not find a version that satisfies the requirement wyze_sdk (from octoprint-wyze)
ERROR: No matching distribution found for wyze_sdk
Error!
Could not parse output from pip, see plugin_pluginmanager_console.log for generated output
```

**Comment by eshapiro42** [comment]
Hi @tigeryell99.

OctoPi 0.18 includes Python 3.7.3 by default. Unfortunately Python >=3.8 is required to install this plugin since (as you're seeing) the wyze_sdk module that it depends on is not available for Python 3.7 or earlier.

If you still want to use it, you'll need to upgrade your OctoPrint's Python environment. Please see the bottom section "Installing Manually" at [this link](https://octoprint.org/download/).

**Comment by tigeryell99** [comment]
Thank you for the quick response.
